server/rpc/dkg_method_handlers.go

```diff
@@ -500,9 +501,33 @@ func (h KeyShareRequestHandler) ServeJSONRPC(c context.Context, params *fastjson
 		if err != nil {
 			return nil, &jsonrpc.Error{Code: -32602, Message: "Internal error", Data: "Error occurred while marshalling" + err.Error()}
 		}
-		verified, userID, err := broker.VerifierMethods().Verify((*bijson.RawMessage)(&redactedRawItem))
+		partitioned, err := tendermint.GetAppKeyPartition(broker, parsedVerifierParams.AppID)
 		if err != nil {
-			return nil, &jsonrpc.Error{Code: -32602, Message: "Internal error", Data: "Error occurred while verifying params" + err.Error()}
+			return nil, &jsonrpc.Error{Code: -32602, Message: "Internal error", Data: "Error occurred while getting a partition" + err.Error()}
+		}
+
+		var verified bool
+		var userID string
+
+		if !partitioned || true {
```

Comment:
The `|| true` part can be removed once the ‘global apps not appearing with partitioned == false’ problem is solved.